sdk/src/timing.rs

```diff
@@ -40,10 +40,12 @@ pub fn years_as_slots(years: f64, tick_duration: &Duration, ticks_per_slot: u64)
 
 /// From slots per year to slot duration
 pub fn slot_duration_from_slots_per_year(slots_per_year: f64) -> Duration {
-    // Regarding division by zero potential below: for some reason, if Rust stores an `inf` f64 and
```

Comment:
@CriesofCarrots This is added by https://github.com/solana-labs/solana/pull/7130/files#diff-826aa0353ee517a070e3c98cc61423eaR43

FYI, rust is actually fixing this behavior: https://github.com/rust-lang/rust/blob/master/RELEASES.md#compatibility-notes

I'm preserving the current behavior.

Reply:
Sweet, thanks!